inference-engine/src/inference_engine/src/ie_core.cpp

```diff
@@ -850,6 +853,9 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                             configCopy[CONFIG_KEY(CACHE_DIR)] = cacheConfig._cacheDir;
                         }
                     }
+                    if (!parser.getDeviceID().empty()) {
+                        configCopy[CONFIG_KEY_INTERNAL(CONFIG_DEVICE_ID)] = parser.getDeviceID();
```

Comment:
what if plugin does not support such key? But supports `DEVICE_ID` like `MYRIAD`?
can we try to enable tests on this platform?

Reply:
Sorry, I don't see that `.c` and `.d` from here CVS-43085 are respected
Does it work for cases if we have not created plugin yet? But trying to set some values via `SetConfig`?